tools/mixxx-isolate.py

```
@@ -0,0 +1,496 @@
+                sys.exit(1)
+            elif helper.returncode is not None:
+                # helper process died
+                log.info(helper.stdout.read(-1))
+                log.error(
+                    " Helper process died: %s."
+                    " Cleanup might be required required",
+                    helper.returncode,
+                )
+                break
+
+            line = helper.stdout.readline().decode("utf-8", "ignore")
+            if line.startswith("STATUS:"):
+                status = line[7:].strip()
+                print("GOT STATUS:" + status)
+                if status == "ISOLATION":
+                    log.info(" Isolation setup complete. Starting mixxx...")
+                    mixxxProc = subprocess.Popen(
+                        [args.mixxx, "--engine-cpu-set", args.cpuset_engine]
+                        + args.mixxx_args
+                    )
+                    mixxxProc.wait()
+                    log.info(" Mixxx finished")
+                    helper.stdin.write("exit\n".encode("ASCII"))
+                    try:
+                        # this might likely not work
+                        helper.send_signal(signal.SIGUSR1)
+                    except PermissionError:
+                        pass
+                    break
+                elif status == "CLEAN":
+                    # cleanup finished.
+                    log.info(" Exit successfully")
+                    sys.exit(0)
+            else:
+                print(line)
+
+        except KeyboardInterrupt:
+            log.info(" Received keyboard interrupt. Stopping")
+            if mixxxProc and mixxxProc.returncode is None:
+                mixxxProc.send_signal(signal.SIGINT)
+            break
+        except Exception as e:
+            log.exception(e)
+    # trigger system cleanup
+    try:
+        # this might likely not work
+        helper.send_signal(signal.SIGUSR1)
+    except PermissionError:
+        pass
```

what is the value of silencing this exception?